### PR TITLE
[Support-33562] Expose hidePresetBar API

### PR DESCRIPTION
**Pull request by darrenchann**
closes NSDK-191

exposed hidePresetBar API on iOS and Android

- added hidePresetBar to iOS
- added hidePresetBar to Android
- added prop to DocumentView.tsx

### CODE REVIEW

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]
missing documentation

**Comment by darrenchann** [comment]
> missing documentation

added documentation to API.md

**Review by sgong-pdftron — APPROVED** [review]

File: ios/RNTPTDocumentView.m [hunk 1]
```diff
@@ -3565,6 +3581,11 @@ - (void)rnt_documentViewControllerTextSearchDidStart:(PTDocumentBaseViewControll
     }
 }
 
+- (BOOL)rnt_documentViewControllerIsPresetBarEnabled:(PTDocumentBaseViewController *)documentViewController
```
Comment (dluco) [inline ios/RNTPTDocumentView.m:3584]: Where does this method come from? I don't see the `PTToolGroupToolbar.presetsEnabled` API being used anywhere so not sure how this works.

Comment (darrenchann) [reply ios/RNTPTDocumentView.m:3584]: changed all naming of hidePresetsBar to presetsToolbarHidden across all 3 files:
https://www.pdftron.com/api/ios/Classes/PTToolGroupToolbar.html#/c:objc(cs)PTToolGroupToolbar(py)presetsToolbarHidden

**Review by dluco — APPROVED** [review]
👍

**Pull request merged**